regression/contracts/assigns_enforce_statics/main.c

```diff
@@ -1,15 +1,15 @@
-#include <assert.h>
-#include <stdlib.h>
-
-static int x;
+static int x = 0;
```

Comment:
Random place for comment: the commit message ...
1) Could make clear that this is about code contracts;
2) Should likely have the "A symbol is ..." all moved into the body of the message;
3) Should likely talk about "called functions" instead of "subfunctions."